tests/sv/climate_HassClimateGetTemperature.yaml

```diff
@@ -3,11 +3,13 @@ tests:
   - sentences:
       - "vad är det för temperatur?"
       - "vilken temperatur är det?"
+      - "vad är temperaturen?"
     intent:
       name: HassClimateGetTemperature
   - sentences:
       - "vad är det för temperatur i vardagsrummet?"
       - "vilken temperatur är det i vardagsrummet?"
+      - "vad är temperaturen i köket?"
```

Comment:
So this doesn't work, the test case is expecting to trigger:
```
   intent:
      name: HassClimateGetTemperature
      slots:
        area: "livingroom"
```
As stated below. But the sentence will trigger `area: "kitchen"` so the testcase fails right now.
So ether change `Köket` to `vardagsrummet` or create a new test case expecting the area of `kitchen`

Reply:
Mea culpa. 🤦‍♂️